### PR TITLE
App Runner: activate

### PR DESCRIPTION
**Pull request by JadenSimon**
Register the commands + telemetry.
I've added a few small fixes and added QuickPick items for when API calls error out. I'll be adding a generic solution for this later, but for now this will work.

Also I noticed that App Runner can get in a _really_ bizarre state where it will list services but not give any operations for them. I confirmed that this is happening with the console as well. So I've added a check for that in the service nodes. I likely never noticed this before with the batch polling stuff since bad data was discarded implicitly as there would be no nodes listening for it. 

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by justinmk3 — APPROVED** [review]

**Comment by JadenSimon** [comment]
Had to add some edge-cases for polling, but everything looks good now.

File: src/apprunner/commands/createService.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    try {
+        const wizard = new CreateAppRunnerServiceWizard(node.region)
+        const result = await wizard.run()
+        if (result === undefined) {
+            telemetryResult = 'Cancelled'
+            return
+        }
+
+        await node.createService(result)
```
Comment (justinmk3) [inline src/apprunner/commands/createService.ts:22]: We probably don't want logic to live in the tree nodes. Instead logic lives in the service client, and values are passed as parameters to the client functions.

The refresh() in `node.createService()` could be done as a `onAfter` callback to the client (continuation-passing-style).

Comment (JadenSimon) [reply src/apprunner/commands/createService.ts:22]: Yeah I think this brings up the question about what the nodes represent within the code. This implementation portrays them as (almost) the 'source of truth' for the corresponding resource. So any caller that wants to interact with App Runner ideally goes through the nodes, not the client. This keep the nodes in-sync as far as mutations within the toolkit go, but obviously would not scale if you had other logic dependent on the resource.

Comment (JadenSimon) [reply src/apprunner/commands/createService.ts:22]: This concept I'm describing is also at odds with current toolkit architecture. Currently, clients are easily accessible while tree nodes are not. Having the node _be_ the resource requires the inverse: nodes are easily accessible while clients are not.

File: src/apprunner/explorer/apprunnerServiceNode.ts [hunk 2]
```diff
@@ -116,16 +117,27 @@ export class AppRunnerServiceNode extends CloudWatchLogsBase implements AWSResou
...
+                const operations = resp.OperationSummaryList
+                const operation = operations && operations[0]?.EndedAt === undefined ? operations[0] : undefined
+                if (operation !== undefined) {
+                    this.setOperation(this._info, operation.Id!, operation.Type as any)
+                }
+            })
+            .catch(err => {
+                // Apparently App Runner can rarely list deleted services with the wrong status
+                getLogger().warn(
+                    `Unable to list operations for service "${this._info.ServiceName}", it will not be shown.`
```
Comment (justinmk3) [inline src/apprunner/explorer/apprunnerServiceNode.ts:132]: nit: unnecessary entroy that "Unable to" is used here, but "Failed to" is used on the next line.

Comment (JadenSimon) [reply src/apprunner/explorer/apprunnerServiceNode.ts]: The wording was wrong in general. If we can't get the operation, the service still shows but can't pause/run/deploy

**Pull request merged**